assignment_1/2_kth_element.py

```diff
@@ -0,0 +1,101 @@
+# Implement an algorithm to find the kth to last element of a singly linked list
+# Magdalena Grodzińska
+
+# linked list implementation
+
+class Node:
+    def __init__(self, data, next_node):
+        self.data = data
+        self.next_node = next_node
+
+    def get_data(self):
```

Comment:
In Python, instance attributes "data" and "next_node" are by convention public so getters like these are not needed.

If you consider "data" and "next_node" private, prepend an underscore, as in:

self._data = data
self._next_node = next_node

In this case, it looks like they could just be "public" though.